<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>378</width>
    <height>255</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tile Cache</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../MBR/Map.ico</normaloff>../MBR/Map.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>160</width>
      <height>132</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="highLatLabel">
       <property name="text">
        <string>High Latitude</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="highLatBox"/>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lowLatBox"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lowLatLabel">
       <property name="text">
        <string>Low Latitude</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="highLongBox"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="highLongLabel">
       <property name="text">
        <string>High Longitude</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="lowLongLabel">
       <property name="text">
        <string>Low Longitude</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="lowLongBox"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="formLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>40</y>
      <width>160</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_2">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <property name="verticalSpacing">
      <number>7</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="dpiLabel">
       <property name="text">
        <string>DPI</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="dpiBox">
       <property name="text">
        <string>96</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="zoomLabel">
       <property name="text">
        <string>Levels of Zoom</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="zoomBox">
       <property name="text">
        <string>20</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="pixLabel">
       <property name="text">
        <string>Tile Width (pixels)</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="pixBox">
       <property name="text">
        <string>512</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Required</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Optional</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="resultsButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>170</y>
      <width>89</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Compute Results</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>378</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTile_Cache">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="resetButton"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuTest">
    <property name="title">
     <string>Test</string>
    </property>
    <addaction name="test10"/>
    <addaction name="testBrazil"/>
    <addaction name="testWorld"/>
    <addaction name="testConus"/>
    <addaction name="testAK"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionJust_Kidding"/>
   </widget>
   <addaction name="menuTile_Cache"/>
   <addaction name="menuTest"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="resetButton">
   <property name="text">
    <string>Reset</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="test10">
   <property name="text">
    <string>10 Level 45 lat 80 long</string>
   </property>
  </action>
  <action name="testBrazil">
   <property name="text">
    <string>Test Brazil</string>
   </property>
  </action>
  <action name="testWorld">
   <property name="text">
    <string>Test World</string>
   </property>
  </action>
  <action name="testConus">
   <property name="text">
    <string>Test Conus</string>
   </property>
  </action>
  <action name="testAK">
   <property name="text">
    <string>Test AK</string>
   </property>
  </action>
  <action name="actionJust_Kidding">
   <property name="text">
    <string>Just Kidding</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>232</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
